### PR TITLE
COMP: Migrate ITKVtkGlue library configuration to itk-module-init.cmake

### PR DESCRIPTION
**Pull request by thewtex**
This enables specifying only the required modules in
test/CMakeLists.txt.

To address #2333

### CODE REVIEW

**Comment by seanm** [comment]
Just tried against vtk master before and after this patch and it indeed fixes the issue!  Much thanks!

File: Modules/Bridge/VtkGlue/CMakeLists.txt [hunk 1]
```diff
@@ -31,54 +31,7 @@ set(CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} ${VTK_REQUIRED_EXE_LINKER_
...
-elseif( ${VTK_VERSION} VERSION_LESS 6.0.0 )
-  set(ITKVtkGlue_VTK_INCLUDE_DIRS ${VTK_INCLUDE_DIRS})
-  set(ITKVtkGlue_VTK_LIBRARIES ${VTK_LIBRARIES})
-else()
-  vtk_module_config(ITKVtkGlue_VTK
-    ${_required_vtk_libraries}
-    )
-  set(ITKVtkGlue_VTK_LIBRARIES ${_required_vtk_libraries})
-endif()
+message("_required_vtk_libraries CMakeLists.txt ${_required_vtk_libraries}")
```
Comment (dzenanz) [inline Modules/Bridge/VtkGlue/CMakeLists.txt:34]: Remove debug code?

File: Modules/Bridge/VtkGlue/test/CMakeLists.txt [hunk 2]
```diff
@@ -18,11 +18,12 @@ if(NOT VTK_RENDERING_BACKEND STREQUAL "None")
     )
 endif()
 
+message("_required_vtk_libraries test/CMakeLists.txt ${_required_vtk_libraries}")
```
Comment (dzenanz) [inline Modules/Bridge/VtkGlue/test/CMakeLists.txt:21]: Remove debug code?

**Review by dzenanz — APPROVED** [review]

File: Modules/Bridge/VtkGlue/CMakeLists.txt (hunk 1 above)
Comment (thewtex) [reply Modules/Bridge/VtkGlue/CMakeLists.txt:34]: :+1:

File: Modules/Bridge/VtkGlue/test/CMakeLists.txt (hunk 2 above)
Comment (thewtex) [reply Modules/Bridge/VtkGlue/test/CMakeLists.txt:21]: :+1:

**Review by dzenanz — APPROVED** [review]

**Comment by thewtex** [comment]
Merging for ITK 5.2rc03.

**Pull request merged**